src/app/loadout/LoadoutDrawerContents.tsx

```diff
@@ -112,6 +117,9 @@ export default function LoadoutDrawerContents(
               <AppIcon icon={addIcon} /> {t('Loadouts.AddEquippedItems')}
             </a>
           )}
+          <a className="dim-button loadout-add" onClick={doFillLoadoutFromInventory}>
+            <AppIcon icon={addIcon} /> {t('Add Unequipped Items')}
```

Comment:
We localize all strings by putting them in `i18n.json` and then referencing the key here. See the button above that uses `t('Loadouts.AddEquippedItems')`